modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc

```diff
@@ -328,9 +328,9 @@ class NDB_Menu_Filter_Form_Resolved_Violations extends NDB_Menu_Filter_Form
         // create user object
         $user        =&        User::singleton();
         $config      =&        NDB_Config::singleton();
-        $study       = $config->getSetting('study');
+        $study       = $config->getSetting('Study');
```

Comment:
Why is this changed to upper case? It's lower case in the schema (and everywhere else in the codebase..)